src/main/java/sc/fiji/bdvpg/scijava/services/BdvSourceAndConverterService.java

```diff
@@ -85,23 +85,23 @@
      * Map containing objects that are 1 to 1 linked to a Source
      * TODO : ask if it should contain a WeakReference to Source keys (Potential Memory leak ?)
      */
-    Map<SourceAndConverter, Map<String, Object>> data;
+    Map<SourceAndConverter, Map<String, Object>> sourceAndConverterToMetadata;
 
     /**
-     * Reserved key for the data map. data.get(sourceandconverter).get(SPIMDATAINFO)
```

Comment:
I'd rather keep 'info' somewhere in the field name because this also contains a reference to the view setup id, and in the future maybe some other stuff (like the original file or a list of successive files)

Reply:
done in https://github.com/bigdataviewer/bigdataviewer-playground/pull/66/commits/114d31a3b4b9b50365f50c2ab2f562c3b915ea00